libpod/container_internal.go

```diff
@@ -1753,6 +1746,13 @@ func (c *Container) mountNamedVolume(v *ContainerNamedVolume, mountpoint string)
 			return vol, nil
 		}
 
+		// Set NeedsCopyUp to false since we are about to do first copy
```

Comment:
Why is this moving up? This could cause us to trap the volume in a loop of attempting to copy up if reading the source to copy up fails.

Reply:
I'm still a bit worried about this one - it seems like we won't set NeedsCopyUp to false if the directory we want to copy up either does not exist or is empty.

Reply:
If this matches Docker it should not a problem.  In the first case you don't want the flag to be set.
If the first time there is an error, you fix the error, now the copyup will not happen, and you have to delete the volume and recreate it.